libstuff/SPerformanceTimer.cpp

```diff
@@ -18,6 +19,12 @@ void SPerformanceTimer::stop() {
     auto now = chrono::steady_clock::now();
     auto duration = now - _lastStart;
 
+    if (_logImmediate) {
+        double durationMS = chrono::duration_cast<chrono::duration<double, std::milli>>(duration).count();
+        SINFO(_description << " " << _lastType << " in " << durationMS << " ms.");
+        return;
```

Comment:
This seems to break the purpose of this class, as we no longer record the totals. Either that or it makes this into a fairly complex way to do a simpler job.

I am confused what the purpose of this change is. I'm guessing you want to use this class somewhere new, but for a simpler task. 

It seems we could do that with a class that looks like:
```
class NewClass {
    chrono::steady_clock::time_point _lastStart = 0;
    void start() {
        _lastStart = 0;
    }
    void stop() {
        SINFO("Message about" << now - _lastStart);
    }
};
```

I wonder if we can collapse SAutoTimer and SPerformanceTimer into one thing, since they seem to do basically the same task, and then make a simpler timer class oif that's what we need.

Reply:
Thanks. I don't have super strong feelings on this so let me know whatever you'd prefer. I am mainly looking for something to time some parts of the Auth code. I originally proposed to add a `Bench` class to Auth, but @coleaeason thought maybe we should use something that already exists.

Reply:
Maybe add the Bench class in auth and create an issue to collapse these two Bedrock classes? It sounds like bench needs less functionality than we have here, and we can probably use only one of the two existing classes in bedrock.